### PR TITLE
Add a prop `quality` for flexible usage

### PR DESCRIPTION
**Pull request by WendellLiu**
# Props
##  `quality`
type: `number`
default value: 1

Besides, I updated the README in this PR .

### CODE REVIEW

**Comment by mozmorris** [comment]
Thanks for this @WendellLiu.

Couple of things:

1. `quality` should be renamed `screenshotQuality`
2. The default value should be set to `0.92`

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ onUserMedia      | function | noop         | callback when component receives a
 onUserMediaError | function | noop         | callback when component cannot receives any media stream width error param
 audioSource      | string   |              | an array or single ConstrainDOMString(s) specifying the device id
 videoSource      | string   |              | an array or single ConstrainDOMString(s) specifying the device id
+quality      | number   |        1      | quality of screenshot(0 to 1)
```
Comment (mozmorris) [inline README.md]: rename `screenshotQuality`

File: src/react-webcam.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ export default class Webcam extends Component {
     onUserMediaError: () => {},
     screenshotFormat: 'image/webp',
     width: 640,
+    quality: 1,
```
Comment (mozmorris) [inline src/react-webcam.js]: set to `0.92`

**Comment by WendellLiu** [comment]
@mozmorris 
I just fix them.

**Comment by mozmorris** [comment]
@WendellLiu looks great, thank you.

**Pull request merged**